maestrowf/interfaces/script/slurmscriptadapter.py

```diff
@@ -200,7 +200,8 @@ def submit(self, step, path, cwd, job_map=None, env=None):
             jid = re.search('[0-9]+', output).group(0)
             return SubmissionRecord(SubmissionCode.OK, retcode, jid)
         else:
-            LOGGER.warning("Submission returned an error.")
+            LOGGER.warning("Submission returned an error (see next line).")
+            LOGGER.warning(output)
```

Comment:
@crkrenn -- Two things:
1. Does `sbatch` really return error messages on `stdout`? It'd be surprised if it ends up on `stdout` normally since `stderr` is returned there by default. Would be worth testing to make sure that `output` here does contain the error if you haven't already. If you have, then just disregard this item. :)
2. It's a knitpick, but it's cleaner to do single log calls:
    ```
    LOGGER.warning("Submission returned an error (see next line).\n%s", output) # or err depending
    ```

Lastly, thank you for contributing. The help is appreciated. 👍

Reply:
I made the changes Frank suggested.

Reply:
Appreciated. The style check is still failing due to some whitespace. Looks like it's on L204. I'll merge it in after that passes.